src/allmydata/scripts/create_node.py

```diff
@@ -180,6 +180,7 @@ class CreateNodeOptions(CreateClientOptions):
     optFlags = [
         ("no-storage", None, "Do not offer storage service to other nodes."),
         ("storage-dir", None, "Path where the storage will be placed."),
+        ("helper", None, "Enable helper"),
```

Comment:
This seems like a distinct feature which could have its own issue/PR and probably warrants a news fragment of its own.

Reply:
https://github.com/tahoe-lafs/tahoe-lafs/pull/640